lib/set-cookie.js

```diff
@@ -20,9 +20,15 @@ function parseString(setCookieValue, options) {
     ? Object.assign({}, defaultParseOptions, options)
     : defaultParseOptions;
 
+  try {
+    value = options.decodeValues ? decodeURIComponent(value) : value; // decode cookie value
+  } catch(e) {
+    console.error(e);
```

Comment:
It might be good to add a bit of context here, something like

```suggestion
    console.error('set-cookie-parser encountered an error decoding a cookie value. Set options.decodeValues to false to disable this feature.', e);
```

Reply:
Included with latest commit.